spec/features/user_views_vendor_list_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+feature 'user views vendor list', %{
+  As visitor
+  I want to view a list of all available food trucks/vendors
+  So that I can decide what food truck/vendors I want to visit
+} do
+  # Acceptance Criteria
+
+  # * As a visitor
+  # * When I visit the home page
+  # * I should be able to click on a link to view all vendors.
+
+  # * As a visitor
+  # * When I visit the vendors page
+  # * I should be able to see the list of vendors.
```

Comment:
These look like more user stories, not acceptance criteria! Acceptance criteria should just be bullet points detailing the core requirements of the functionality. I.e.:

```
- All vendors should be visible on this page
- I should be able to get to this page via clicking on a link from the home page
```